vdb-bench-assembly/plugins/vdb-bench-dataservice/dsPageService.js

```diff
@@ -167,8 +167,8 @@
         };
         pages[service.SERVICESOURCE_NEW_PAGE] = {
             id: service.SERVICESOURCE_NEW_PAGE,
-            title: 'New Service-source',
-            icon: 'fa-plus',
+            title: 'Configure Source',
```

Comment:
Verbage on how to describe what  we're doing under the hood to connect to a temporary, source-only VDB and pull back Teiid-specific schema will **always** be difficult.  Creating **New** things isn't what this is about, so Randall suggested and the concensus was to use **Configure** for the time being and see how it lands. 

For JDBC sources, configuring means tweaking the connection props (filtering) and exposing the source tables I wish to use in their views

For other sources (file, xml, etc) configuring means presenting the available metadata and helping users defined the tables they wish to use in their views.